### PR TITLE
soc/qdu1000: add status for QDU1000/QRU1000

### PR DESCRIPTION
**Pull request by Komal-Bajaj**
QDU1000/QRU1000 Qualcomm platforms are supported by upstream. Describe its status.

### CODE REVIEW

File: _pmic/pm8150.md [hunk 1]
```diff
@@ -30,3 +30,4 @@ pmic-usb-typecpd: N/A
 pmic-watchdog: N/A
 pmic-wled: N/A
 ---
+This PMIC is usually used with the [QDU1000/QRU1000](../soc/qdu1000) platform.
```
Comment (lumag) [inline _pmic/pm8150.md]: Not only. Please also list sm8150 and sm8250

File: _soc/qdu1000.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+status-connectivity-ipa: N/A
+status-connectivity-wlan: N/A
+status-cpu-bwmon:
+status-cpu-cachetop:
+status-cpu-cpufreq: 6.3
+status-cpu-cpuidle: 6.3
+status-cpu-ddrfreq:
+status-cpu-l3cache:
+status-cpu-llcc: WIP
+status-cpu-smp:
```
Comment (lumag) [inline _soc/qdu1000.md]: SMP configuration is already present, so could you please fix this.

File: _soc/qdu1000.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+status-storage-sdcc: WIP
+status-storage-ufs: N/A
+status-storage-ufsice: N/A
+status-uart: 6.3
+status-usb: WIP
+status-usb-hostmode:
+status-usb-periphmode:
+status-usb-typec:
+status-usb-usb4:
+status-usb-usbotg:
```
Comment (lumag) [inline _soc/qdu1000.md]: Please add N/A to USB modes, where applicable

**Review by lumag — CHANGES_REQUESTED** [review]
It mostly looks good, could you please fix the small notes.

File: _pmic/pm8150.md (hunk 1 above)
Comment (Komal-Bajaj) [reply _pmic/pm8150.md]: Okay, will add it.

File: _soc/qdu1000.md (hunk 2 above)
Comment (Komal-Bajaj) [reply _soc/qdu1000.md]: Okay, will fix it.

File: _soc/qdu1000.md (hunk 3 above)
Comment (Komal-Bajaj) [reply _soc/qdu1000.md]: Okay, will update it.

**Review by lumag — APPROVED** [review]

**Comment by lumag** [comment]
Thank you, merging now.

**Pull request merged**